src/app/data-planes/components/DataPlaneDetails.vue

```diff
@@ -0,0 +1,96 @@
+<template>
+  <div class="columns">
```

Comment:
I think the assumption that details on the tray and details in the detail view will be the same is wrong. We should probably have different views.